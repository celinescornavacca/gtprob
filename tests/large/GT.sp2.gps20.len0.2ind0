((((A1,((A10,A11),(A18,A19))),((A2,A3),(A4,(A14,A15)))),(((A6,(A7,A8)),(A0,(((A16,A17),A9),A5))),(A12,A13))),((B6,B7),(((B1,B2),(B18,B19)),(((B3,B4),(B11,B12)),((((B14,B15),B13),B5),(B0,((B10,(B16,B17)),(B8,B9))))))));
