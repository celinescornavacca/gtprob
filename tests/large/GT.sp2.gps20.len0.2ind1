(((A3,A4),(((A5,A6),((A10,A11),A9)),((((((A12,(A18,A19)),(A13,A14)),((A16,A17),A15)),A1),A0),((A7,A8),A2)))),((((B1,(B2,(B18,B19))),(B16,B17)),((B7,B8),(((B4,B5),((B9,B10),(B6,(B11,B12)))),B0))),(B3,(B13,(B14,B15)))));
