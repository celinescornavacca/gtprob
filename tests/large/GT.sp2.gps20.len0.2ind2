(((A1,(A9,(A14,A15))),((((A12,A13),A3),(A0,(((A18,A19),A7),A2))),((((A16,A17),A6),(A8,(A10,A11))),(A4,A5)))),(((B0,((B9,B10),((B13,B14),B12))),(B1,((B3,(B5,((B18,B19),B11))),((B7,B8),B4)))),(B2,(B6,((B16,B17),B15)))));
