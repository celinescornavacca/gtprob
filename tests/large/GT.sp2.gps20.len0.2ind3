(((((((A6,A7),((A10,(A18,A19)),A4)),A1),(A5,((A14,A15),A8))),A0),(((A3,((A12,A13),A11)),A2),((A16,A17),A9))),(((((B7,(B11,B12)),B6),((B3,B4),(B9,B10))),(B1,(B2,((B15,((B18,B19),(B16,B17))),B8)))),(B0,((B13,B14),B5))));
