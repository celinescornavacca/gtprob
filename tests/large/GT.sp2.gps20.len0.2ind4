(((A11,A12),((A6,A7),((((A13,A14),A2),(A4,A5)),((A8,A9),(A0,(((A3,(A15,A16)),(A10,(A17,(A18,A19)))),A1)))))),((((((B2,(((B7,B8),B5),(B6,((B16,B17),B9)))),(B3,B4)),B1),(((B18,B19),B12),(B13,(B14,B15)))),B0),(B10,B11)));
