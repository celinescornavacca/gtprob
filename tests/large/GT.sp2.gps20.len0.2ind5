(((((((A11,(((A18,A19),A17),A16)),A10),A4),(A12,A13)),(A2,A3)),((A8,A9),(((A14,A15),A0),(A1,(A5,(A6,A7)))))),((((((B16,B17),(B4,(B5,B6))),(B18,B19)),((((B12,B13),((B14,B15),B11)),(B9,B10)),(B7,B8))),(B0,B1)),(B2,B3)));
