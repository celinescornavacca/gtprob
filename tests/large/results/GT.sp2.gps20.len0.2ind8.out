(((((A2,A3),(A5